src/utils/eventLevels.js

```diff
@@ -55,17 +55,14 @@ export function eventLevels(rowSegments, limit = Infinity) {
   return { levels, extra }
 }
 
-export function inRange(e, start, end, accessors) {
-  let eStart = dates.startOf(accessors.start(e), 'day')
-  let eEnd = accessors.end(e)
+export function inRange(e, lowerBound, upperBound, accessors) {
```

Comment:
can you add a comment noting the performance concerns of this method as an explanation for why it does use `date-arithmetic`, so someone doesn't accidentally remove the code later :P

Reply:
Commented the inRange method